Makefile

```diff
@@ -14,6 +14,10 @@ end-to-end-test:
 	sudo docker build -f ./test/end-to-end/Dockerfile -t weld/end-to-end:latest ./test/end-to-end/
 
 	make -C welder-deployment/ repos
+	git clone ../welder-web
```

Comment:
make repos will clone welder-web code from master into welder-deployment, that'll replace the repo we just cloned.
so we must make repos first, and delete welder-web repo which is cloned from master.
then, clone welder-web from local folder which contains the PR code.